pyiode/iode_database/variables_database.pyx

```diff
@@ -961,6 +975,7 @@ cdef class Variables(_AbstractDatabase):
             filepath to the input file.
         lambda_: float
             ???
+        # GDM> why not variables?
```

Comment:
Because the algorithm of the seasonal_adjustment() method is based on the `Census XI method` which applies on "series". Also the French documentation of the IODE command `WsSeasonAdj` refers to "series"